lib/modules/dosomething/dosomething_kudos/dosomething_kudos.module

```diff
@@ -325,12 +325,10 @@ function dosomething_kudos_get_kudos_data_for_fid($fid = NULL, $total_rogue_kudo
     return [];
   }
 
-  if ($total_rogue_kudos === 0) {
-    $total_kudos = 0;
-  } elseif ($total_rogue_kudos) {
-    $total_kudos = $total_rogue_kudos;
-  } else {
+  if(!isset($total_rogue_kudos)) {
```

Comment:
oh @DFurnes I like that a lot better!! I just updated it - do you mind taking another look and giving comments or a 👍 ?